test/units/modules/network/aci/test_aci_rest.py

```diff
@@ -0,0 +1,299 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017 Dag Wieers <dag@wieers.com>
+
+# This file is part of Ansible by Red Hat
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import sys
+
+from ansible.compat.tests import unittest
+from ansible.modules.network.aci.aci_rest import aci_response
+
+from nose.plugins.skip import SkipTest
+
+from lxml import etree
+if sys.version_info >= (2, 7):
+    from xmljson import cobra
+
+
+class AciRest(unittest.TestCase):
+
+    def test_invalid_aci_login(self):
+        self.maxDiff = None
+
+        expected_result = {
```

Comment:
my personal preference is to stuff things like this into a fixture file and then load them using something like load_json method that I found (at one time, dunno if it's there anymore) in the nxos module unit tests.

Here's what it looked like at the time
https://github.com/ansible/ansible/blob/devel/test/units/modules/network/f5/test_bigip_hostname.py#L58

and then stuff the JSON in a `fixtures/` directory